mws/resource_mws_pas_test.go

```diff
@@ -0,0 +1,246 @@
+package mws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMWSPAS(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test in short mode.")
+	}
+	acctID := qa.GetEnvOrSkipTest(t, "DATABRICKS_ACCOUNT_ID")
+	client := common.CommonEnvironmentClient()
+	ctx := context.Background()
+	pasAPI := NewPrivateAccessSettingsAPI(ctx, client)
+	pasList, err := pasAPI.List(acctID)
+	assert.NoError(t, err, err)
+	t.Log(pasList)
+
+	pas := PrivateAccessSettings{
+		AccountID: acctID,
+		PasName:   qa.RandomName(),
+		PasID:     "",
```

Comment:
probably remove `PasID` line, it's not needed, as this field is computed

Reply:
Done